src/pokedex.c

```diff
@@ -14,6 +14,9 @@
 #include "overworld.h"
 #include "palette.h"
 #include "pokedex.h"
+#if POKEDEX_PLUS_HGSS == TRUE
+#include "pokedex_plus_hgss.h"
```

Comment:
`include/pokedex_plus_hgss.h` is already cleared out when `POKEDEX_PLUS_HGSS` is false, so there's no need to hide this include as well.